libstb/secvar/storage/secboot_tpm.c

```diff
@@ -370,8 +371,17 @@ static int secboot_tpm_store_init(void)
 
 	prlog(PR_DEBUG, "Initializing for pnor+tpm based platform\n");
 
-	// TODO: check physical presence here
-	// TODO: undefine indices if necessary here
+	/* Check if physical presence is asserted */
+	if (secvar_check_physical_presence()) {
+		prlog(PR_INFO, "Physical presence asserted, redefining NV indices, and resetting keystore\n");
+		rc = tss_nv_undefine_space(SECBOOT_TPMNV_VARS_INDEX);
```

Comment:
This is doing direct call to tss_nv_undefine_space() unlike other TSS calls which are done as hooks of tpmnv_ops.